packages/webdriverio/src/utils/index.ts

```diff
@@ -0,0 +1,105 @@
+import type { BrowserObject } from 'webdriverio';
+import type { AxeResults, ElementContext } from 'axe-core';
+import type {
+  AnalyzePageParams,
+  AnalyzePageResponse,
+  DoneFunction
+} from '../types';
+
+/**
+ * Validates that the client provided is WebdriverIO v5 or v6.
+ * @param {any} client
+ * @returns {BrowserObject}
+ */
+
+export const isWebdriverClient = (client: any): client is BrowserObject => {
+  if (!client || typeof client !== 'object') {
+    return false;
+  }
+
+  if (typeof client.execute !== 'function') {
+    return false;
+  }
+
+  if (typeof client.switchToFrame !== 'function') {
+    return false;
+  }
+
+  return true;
+};
+
+/**
+ * Get running context
+ * @param {Array} include
+ * @param {Array} exclude
+ * @returns {(ElementContext | null)}
+ */
+
+export const normalizeContext = (
+  include: string[],
+  exclude: string[]
+): ElementContext | null => {
+  if (!exclude.length) {
+    if (!include.length) {
+      return null;
+    }
+    return {
+      include
+    };
+  }
+  if (!include.length) {
+    return {
+      exclude
+    };
+  }
+  return {
+    include,
+    exclude
+  };
+};
+
+/**
+ * Analyze the page.
+ * @param {AnalyzePageParams} analyzeContext
+ * @param {DoneFunction} done
+ * @returns {AnalyzePageResponse}
+ */
+
+export const analyzePage = (
+  analyzeContext: AnalyzePageParams,
+  done: DoneFunction
+): AnalyzePageResponse => {
+  const axeCore = (window as any).axe;
```

Comment:
You should do this - https://github.com/dequelabs/axe-devtools-npm/blob/develop/packages/cypress/src/types.ts#L15